HotTips/TipObjects/TipObjects.cs

```diff
@@ -68,6 +68,31 @@ public static string GetTipId(string globalTipId)
         }
     }
 
+    public class TipHistoryInfo
+    {
+        public string globalTipId { get; set; }
+
+        public TipLikeEnum tipLikeStatus { get; set; }
+
+        public override string ToString()
```

Comment:
This is clever, but perhaps not the most correct use ToString. Might be better if this was named differently like "Serialize"